include/robot_interfaces/sensors/pylon_driver.hpp

```diff
@@ -33,58 +33,71 @@ class PylonDriver : public SensorDriver<CameraObservation>
     Pylon::CTlFactory& tl_factory_;
     Pylon::CImageFormatConverter format_converter_;
     Pylon::CPylonImage pylon_image_;
-    const std::string& device_user_id_to_open = "cam_1";
+    const std::string& device_user_id_to_open_;
```

Comment:
These variables are all only used inside single functions, so I think they could be defined locally.

Reply:
but they are being called by the constructor, so shouldn't they be defined as class variables?

Reply:
Not sure if I am missing something, but const and reference members cannot be initialized in the constructor, right?